nuget/runners/nunit.runners.nuspec

```diff
@@ -28,7 +28,7 @@
     <releaseNotes>This release uses the latest version of the TeamCityEventListener extension.</releaseNotes>
     <language>en-US</language>
     <tags>nunit test testing tdd runner</tags>
-    <copyright>Copyright (c) 2017 Charlie Poole</copyright>
+    <copyright>Copyright (c) 2018 Charlie Poole</copyright>
```

Comment:
Is it on purpose that this file as the only .nuspec file that only contains Charlie's name?

Reply:
I think I left it as is because I wasn't going to update it any longer, just have it depend on NUnit.Console latest version. Rob decided to keep releasing it, so it should be updated like the others.